substra/__init__.py

```diff
@@ -12,12 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from substra.__version__ import __version__
```

Comment:
This is unrelated to the API version. This is the version of the package and should not be removed.

Reply:
Oops, I forgot to put it back when I fixed jmorel's comments.